{{cookiecutter.project_slug}}/Makefile

```diff
@@ -179,10 +179,10 @@ filebrowser:  ### Run filebrowser job
 		filebrowser/filebrowser
 
 .PHONY: kill-filebrowser
-kill-filebrowser:  ### Kill filebrowser job
+kill-filebrowser:  ### Terminate a job with File Browser
 	neuro kill $(FILEBROWSER_NAME)
 
-.PHONY: kill  ### Kill training, jupyter, tensorboard and filebrowser jobs (even if some of them are not running)
+.PHONY: kill  ### Terminate all jobs of this project
 kill: kill-training kill-jupyter kill-tensorboard kill-filebrowser
```

Comment:
By the way: let's change this command to `neuro kill $(neuro --quiet ps)`? I think it's good to teach our users this cool command :+1:

Reply:
Nope. The command is cool, and I describe it at  https://neu.ro/docs/faq#how-to-kill-all-running-jobs. But here you don't want to kill all of your jobs, only the ones related to this project.